prow/config.yaml

```diff
@@ -1247,7 +1247,7 @@ presubmits:
     trigger: "(?m)^/test( all| pull-test-infra-bazel),?(\\s+|$)"
     spec:
       containers:
-      - image: gcr.io/k8s-testimages/bazelbuild:v20170925-2204c0cb
+      - image: gcr.io/k8s-testimages/v20170929-ed26bd34-0.6.0
```

Comment:
+bazelbuild?

Reply:
grr how did I...
`gcr.io/k8s-testimages/bazelbuild:v20170929-ed26bd34-0.6.0` is what it should be, fixing. Thanks..

Reply:
done + squashed